swan-lake/learn-the-platform/source-code-dependencies/manage-dependencies.md

```diff
@@ -117,6 +117,10 @@ name = "mysql"
 version = "1.5.0”
 ```
 
+**Building with a new distribution version**
```

Comment:
Wrote the following with the help of ChatGPT 

### Upgrading packages to use new Swan Lake Update distributions

When a new Swan Lake Update distribution is released, it may include incompatible language improvements. When upgrading an existing package to use new Swan Lake Update distributions, the Ballerina compiler automatically updates the `Dependencies.toml` file with the latest compatible minor versions of dependencies for the new distribution. This enables existing projects to benefit from improvements in the new Update without significant modifications.